### PR TITLE
Linting and fixing #475

### PR DESCRIPTION
**Pull request by AkosLukacs**

### CODE REVIEW

File: devices/BMP280.js [hunk 1]
```diff
@@ -33,18 +33,19 @@ function BMP280(options, read, write) {
...
-  var r = this.read(0xF4,1)[0]; // ctrl_meas_reg
-  if (on) r |= 3; // normal mode
-  else r &= ~3; // sleep mode
-};
+// TODO: this does nothing? (same in BME280 and BMP280)
+// BMP280.prototype.setPower = function(on) {
+//   var r = this.read(0xF4,1)[0]; // ctrl_meas_reg
+//   if (on) r |= 3; // normal mode
+//   else r &= ~3; // sleep mode
+// }
```
Comment (gfwilliams) [inline devices/BMP280.js]: I think we actually just need a `this.write(0xF4,r)` here instead

**Comment by gfwilliams** [comment]
Looks great - I'm up for merging this now if you are?

**Comment by AkosLukacs** [comment]
Yes!

**Comment by gfwilliams** [comment]
Thanks - you must have spent ages on this!

**Comment by AkosLukacs** [comment]
Just slightly more than originally estimated :)

**Pull request merged**